src/PrintLoopNest.cpp

```diff
@@ -197,5 +198,19 @@ string print_loop_nest(const vector<Function> &output_funcs) {
     return sstr.str();
 }
 
+string print_loop_nest(const vector<Function> &output_funcs) {
+    // For the purposes of printing the loop nest, we don't want to
+    // worry about which features are and aren't enabled.
+    Target target = get_host_target();
+    for (DeviceAPI api : all_device_apis) {
+        target.set_feature(target_feature_for_device_api(DeviceAPI(api)));
+    }
+    // target_feature_for_device_api doesn't handle DeviceAPI::Hexagon becuase
```

Comment:
This is a bit of a hack. Probably better to change target_feature_for_device_api to return HVX_128 for hexagon.